src/topology/algebra/infinite_sum.lean

```diff
@@ -601,6 +602,44 @@ end tsum
 
 end topological_semiring
 
+section topological_semimodule
+variables {R : Type*}
+[semiring R] [topological_space R]
+[topological_space α] [add_comm_monoid α]
+[semimodule R α] [topological_semimodule R α]
+{f : β → α}
+
+lemma has_sum.smul {a : α} {r : R} (hf : has_sum f a) : has_sum (λ z, r • f z) (r • a) :=
+has_sum.map hf ⟨λ y, r • y, smul_zero _, smul_add _⟩ (continuous.smul continuous_const continuous_id)
+
+lemma summable.smul {r : R} (hf : summable f) : summable (λ z, r • f z) :=
+(hf.has_sum.smul).summable
+
+lemma tsum_smul [t2_space α] {r : R} (hf : summable f) : (∑' z, r • f z) = r • (∑' z, f z) :=
+hf.has_sum.smul.tsum_eq
+
+end topological_semimodule
+
+section continuous_linear_map
+variables {R : Type*}
+[semiring R]
+[topological_space α] [add_comm_monoid α]
+[semimodule R α]
+[topological_space γ] [add_comm_monoid γ] [semimodule R γ]
+{f : β → α}
+
+lemma has_sum.map_linear {a : α} (hf : has_sum f a) (g : α →L[R] γ) : has_sum (g ∘ f) (g a) :=
+has_sum.map hf g.to_linear_map.to_add_monoid_hom g.continuous
```

Comment:
See [`continuous_linear_map.has_sum`](https://leanprover-community.github.io/mathlib_docs/analysis/normed_space/operator_norm.html#continuous_linear_map.has_sum) and the next few lemmas. You may want to move them and/or add aliases using `alias` command but please avoid code duplication.

Reply:
Thanks! Sorry I didn't see those before; I deleted the two duplicated lemmas and moved `map_tsum` to `operator_norm.lean`.